packages/canon/css/components.css

```diff
@@ -94,6 +94,36 @@
   height: 1.5rem;
 }
 
+.canon-DataTableRoot {
+  gap: var(--canon-space-3);
+  flex-direction: column;
+  display: flex;
+}
+
+.canon-DataTablePagination {
+  padding-top: var(--canon-space-5);
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+}
+
+.canon-DataTablePagination--left {
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+}
+
+.canon-DataTablePagination--right {
+  justify-content: space-between;
+  align-items: center;
+  gap: var(--canon-space-2);
+  display: flex;
+}
+
+.canon-DataTablePagination--select {
+  min-width: 168px;
```

Comment:
we discussed this a while back where we wanted to use primarly the `rem` unit, any reason for using px here instead?